app/models/comment.rb

```diff
@@ -25,4 +27,16 @@ def self.attributes
       'updated_at'
     ]
   end
+
+  # All of the users who should be notified when a comment is created
+  def listeners
+    users_to_notify = []
+    users_to_notify += self.proposal.currently_awaiting_approvers
+    users_to_notify += self.proposal.approvals.approved.map(&:user)
```

Comment:
No, `currently_awaiting_approvers` only includes approvers which can take action _right now_. That means any approvers later in the chain aren't included.